ruff.toml

```diff
@@ -0,0 +1,56 @@
+exclude = [
```

Comment:
❓ Just curious: is it better to put configuration in a dedicated file, instead of the existing `pyproject.toml`? Looks like Ruff can be configured with one or the other...

Reply:
I concur with @gmarabout at this point.
The less files we have the clearer everything is.

Reply:
It can be merged into the `pyproject.yml`. Will do